### DISCUSSION TITLE
Membership request

### DISCUSSION THREAD

**Discussion by spencerjanssen** [haskell-github-trust/.github#10] [General]
Hello!

I'd like to become a Trust Owner.  I've asked the maintainers of smtp-mail for permission to move that package to the trust (see https://github.com/jhickner/smtp-mail/pull/41#issuecomment-2012521041).  My Hackage account is SpencerJanssen.

**Comment by jamesdbrock** [comment]
Invite sent! Welcome!

Please set your visiblity to Public https://github.com/haskell-github-trust#how-to-add-a-trust-owner

**Comment by jamesdbrock** [comment]
Hi @spencerjanssen Don't forget to add the https://hackage.haskell.org/user/haskell_github_trust account to the smtp-mail Maintainers https://hackage.haskell.org/package/smtp-mail/maintainers/ so that other Trust Owners can also maintain this package.
